frontend/src/app/patients/UploadPatients.tsx

```diff
@@ -363,24 +364,26 @@ const UploadPatients = () => {
                   </p>
                 </div>
                 <a
-                  href="/using-simplereport/manage-people-you-test/bulk-upload-patients/#preparing-your-spreadsheet-data"
                   className={"usa-button margin-right-105"}
-                  onClick={() =>
+                  onClick={() => {
                     appInsights?.trackEvent({
                       name: "viewPatientBulkUploadGuide",
-                    })
-                  }
+                    });
+                    navigate(
+                      "/using-simplereport/manage-people-you-test/bulk-upload-patients/#preparing-your-spreadsheet-data"
+                    );
+                  }}
```

Comment:
these changes address the following error that doesnt cause the test to fail 
![image](https://github.com/CDCgov/prime-simplereport/assets/4952042/8bb91751-7326-413d-868f-d3509435b903)

basically there is a library that is supposed to remove these errors, https://github.com/CDCgov/prime-simplereport/blob/f9422dcf015c2e415b194caf43632e4718f97394/frontend/src/setupTests.js#L8

but I dont believe this is working anymore

Reply:
I ended up removing it because removing this change because removing `href` might introduce accessibility violations